app/src/main/java/com/tsu/calculator/MainModel.kt

```diff
@@ -1,35 +1,185 @@
 package com.tsu.calculator
 
-object MainModel {
-    fun calculatePercent(first: Double): Double {
-        return first / 100.0
+import android.util.Log
+
+class MainModel {
+    private var commaAdded = false
+
+    private var _firstOperand: String = ""
+    private var _secondOperand: String = ""
+    private var _operation: Buttons? = null
```

Comment:
In kotlin we use symbol _ in private variables only if it have val getter. It's not critical but better to remove it
example: 

> private var _number: Int = 0
> val number: Int
>     get() = _number